docs/users/credentials/add-gcp-credential.md

```diff
@@ -15,7 +15,7 @@ links:
 
 1. Access a GCP project or create a new one: [https://console.cloud.google.com/project](https://console.cloud.google.com/project)
     > If you created a new project, a notification modal will appear showing your new project creation status.
-    > Reload the console page if your new project is not listed a few seconds after its creation is succeded.
+    > Reload the console page if your new project is not listed a few seconds after its creation is succeeded.
```

Comment:
The previous section starts with the conditional statement (`If you created a new project`), and only then says what needs to be done in such case?

Shouldn't we adopt the same approach for this sentence?

```suggestion
    > Note: It might take a few seconds for the new project to be visible in the project list.
    > If your new project is not listed a few seconds after its creation, reload the console page.
```

Reply:
Agree!